docs/js/extra.js

```diff
@@ -26,6 +26,8 @@ function setCopyText() {
     'button[data-clipboard-target$="code"]',
   );
   const observer = new IntersectionObserver((entries) => {
+    const start = performance.now();
+
```

Comment:
Should this be removed before merging, or is it intentional?

Reply:
Oops no. I'm sorry. That's absolutely not intentional